### PR TITLE
David/latest qw5 gold characterization

### PR DESCRIPTION
**Pull request by DavidSarlle**
THE PR SHOULD BE MERGED AS SOON AS WE MERGE INTO MAIN QIBOLAB PR 
[alvaro/autocalibration_qblox9_sweepers_refactor_ports
](https://github.com/qiboteam/qibolab/pull/518)

This PR works with latest version of qibocal and qblox drivers. The qw5_gold runcard is optimized for single qubit gates (also the qw5_gold_qblox.py contains some modifications that affects the single qubit performance). Please note that the q4 has a weird (but consistent) behaviour due to the chip itself, so no better performance can be reached.

Reached assigment fidelities per qubit:
q0: 88 - 89%
q1: 90 - 91%
q2: 90 - 91%
q3: 90 - 91%
q4: 80 - 82%

[classify qw5q_gold.pdf](https://github.com/qiboteam/qibolab_platforms_qrc/files/12129490/classify.qw5q_gold.pdf)

Work with branches:
qibolab: https://github.com/qiboteam/qibolab/tree/alvaro/autocalibration_qblox9_sweepers_refactor_ports
qibocal: https://github.com/qiboteam/qibocal/tree/two-qubit-gates

@andrea-pasquale @igres26 @aorgazf @stavros11 @scarrazza @Jacfomg @rodolfocarobene use this runcard as it is the most updated one

### CODE REVIEW

**Comment by DavidSarlle** [comment]
Correction in iq_angle and thresholds introduced

File: qw5q_gold.yml [hunk 1]
```diff
@@ -38,195 +36,228 @@ native_gates:
...
             # parameters for single shot classification
-            iq_angle: 99.758
-            threshold: 0.003933
+            iq_angle: 1.2988731608657835
+            threshold: 0.004916984424451532
 
             # state0_voltage: 0.0
             # state1_voltage: 0.0
             # mean_gnd_states: (-0.0+0.0j)
             # mean_exc_states: (0.0+0.0j)
```
Comment (andrea-pasquale) [inline qw5q_gold.yml:188]: Remove dead code

File: qw5q_gold.yml [hunk 2]
```diff
@@ -38,195 +36,228 @@ native_gates:
...
+            #   type: qf
+            # - duration: 20
+            #   amplitude: 0
+            #   shape: Rectangular())
+            #   qubit: 2
+            #   relative_start: 32
+            #   type: qf
+            # - type: virtual_z
+            #   phase: -0.041
+            #   qubit: 2
```
Comment (andrea-pasquale) [inline qw5q_gold.yml:156]: Remove dead code

**Review by andrea-pasquale — APPROVED** [review]
Thanks @DavidSarlle

File: qw5q_gold.yml (hunk 1 above)
Comment (DavidSarlle) [reply qw5q_gold.yml]: done

File: qw5q_gold.yml (hunk 2 above)
Comment (DavidSarlle) [reply qw5q_gold.yml]: done

**Comment by andrea-pasquale** [comment]
@scarrazza this is ready to merge

**Review by rodolfocarobene — APPROVED** [review]

**Comment by andrea-pasquale** [comment]
Small adjustements. Here are the latest results.
http://login.qrccluster.com:9000/bnkazSHATkazr2pjXGrykQ==

**Pull request merged**